docs/src/operations/requirements.md

```diff
@@ -44,7 +44,7 @@ Docker's containerization overhead and resultant performance degradation unless
 specially configured.
 
 We use Docker only for development purposes. Docker Hub contains images for all
-releases at [solanalabs/solana](https://hub.docker.com/r/solanalabs/solana).
+releases at [agavexyz/agave](https://hub.docker.com/r/agavexyz/agave).
```

Comment:
> i'd be tempted to delete the link from this particular page. maybe the whole paragraph about where we use docker

Didn't notice this in my first pass, but I agree. This page is `Requirements` under the `Operating a Validator` section: https://docs.anza.xyz/operations/requirements

Since we do NOT recommend using Docker to run production validators, there is no reason to mention Docker on the validator page

Reply:
i _do_ think we should keep the no support warning and other discouragements in the first paragraph of this section

Reply:
That works for me. My thought was that even mentioning Docker on this page would plant the idea in someone's head that it is an option they may pursue.

But to your point, keeping the warning/discouragement around gives us something we can point at if someone says "I found your Docker images, but my validator doesn't work when running them"